### ISSUE TITLE
Pedestrians locked out their animation

### ISSUE THREAD

**Issue by burnout2k** [lgsvl/simulator#1860]
Hello

Every now and then some pedestrians are locked to their default stance when it comes to their animation.
Here is an example:
https://youtu.be/UlPL-JLI-NM

It also happens when I run the original maps with original vehicles using your latest build.

Just something to look into moving forward...

**Comment by EricBoiseLGSVL** [comment]
Yes, we are looking into this.  We hope to have a solution for the next release.  Thanks for posting.
